bin/migration/main_script.py

```diff
@@ -22,10 +22,12 @@
 from tables.audits import AuditLogManager
 
 from tables.helpers import clear_migrations_file
+from summary import RecordCounter
+from logger import FailedImportsLogger
 
 
 # get passwords from env variables
-source_db_password = os.environ.get('SOURCE_DB_PASSWORD')
+demo_db_password = os.environ.get('DEMO_DB_PASSWORD')
 destination_db_password = os.environ.get('DESTINATION_DB_PASSWORD')
 test_db_password = os.environ.get('TEST_DB_PASSWORD')
 staging_db_password = os.environ.get('STAGING_DB_PASSWORD')
```

Comment:
Feel free to ignore this for this PR and make it in a follow up PR:

I think for this section, let's simplify it so that you have only source and destination for all of the credentials needed. Which for the source for 
```suggestion
source_db_password = os.environ.get('SOURCE_DB_PASSWORD')
source_db_user = os.environ.get('SOURCE_DB_USER')
source_db_name = os.environ.get('SOURCE_DB_NAME')
source_db_host = os.environ.get('SOURCE_DB_HOST')

destination_db_password = os.environ.get('DESTINATION_DB_PASSWORD')
destination_db_user = os.environ.get('DESTINATION_DB_USER')
destination_db_name = os.environ.get('DESTINATION_DB_NAME')
destination_db_host = os.environ.get('DESTINATION_DB_HOST')
```

This way the mention of test/staging/demo/prod is removed from the code and will be controlled by the environment variables being passed into the script at run time instead.

To make it easier you could create a `.env-<env>` file for each set of variables and commit these (apart from the password ones obviously ;) ) Then for the runner of the script it's actually quite trivial to swap out which credentials are used for a source and a destination.

Reply:
Agreed - way cleaner - will commit this change